### ISSUE TITLE
collections module not updated correctly on import

### ISSUE THREAD

**Issue by elstoc** [darktable-org/darktable#11149]
See the following video for details:

https://user-images.githubusercontent.com/9555491/154283460-c01e6084-5f43-4975-b8fd-435b75210c76.mp4

Basically, I would expect that, after importing, the collect module should filter only those film rolls that were imported (the base folder and any subfolders). It seems to update the collection filter criteria correctly but doesn't filter the contents of the panel. If I update the criteria (delete the last character and re-enter) it works as expected.

**Comment by phweyland** [comment]
Like a collect UI refresh ?

**Comment by elstoc** [comment]
right

**Comment by phweyland** [comment]
Strange, it works for me.

**Comment by elstoc** [comment]
The images are shown but the UI of the collect module includes other film rolls than the ones imported.

**Comment by phweyland** [comment]
Got it! Thanks

**Comment by phweyland** [comment]
Back to look at this.
After importing the right film roll is selected and the corresponding images are shown. This sounds right to me.

You request to filter the film rolls list to the imported one(s). This is equivalent to the manual entry filtering, i.e. when you reduce the visible list to select the right one (which is already done in the issue case). I've tried to go this way and I've created another issue at dt start.

For me that's more the list filtering behavior which should be consistent, i.e. scroll to and highlight the first matching item(s) without hiding the others. But that is another topic probably...

I would close this particular issue.
